atd-vzv/src/views/summary/FatalitiesByTimeOfDay.js

```diff
@@ -129,85 +130,95 @@ const FatalitiesByTimeOfDayWeek = () => {
   return (
     <Container>
       <Row>
-        <h3>Traffic fatalities by time and day</h3>
+        <Col md="2"></Col>
```

Comment:
These two empty `Col`s serve as placeholders to center the `Nav`, right? You could remove them and add the `justify-content-center` class to the Row to do the same thing and make it easier to see what is happening from the code. https://getbootstrap.com/docs/4.0/layout/grid/#horizontal-alignment

Reply:
yeah, we shouldn't have theses tabs stacking like the currently are on certain screen widths...
![Screen Shot 2020-01-14 at 11 26 32 AM](https://user-images.githubusercontent.com/5697474/72366959-ee6d9080-36c0-11ea-8a1b-eadf9fb63fe1.png)

I would make the Col that contains the Nav have a full 12 cell width and center it with bootstrap utilities just like Mike suggests.

Reply:
This is so much better - thanks!